scripts/generate-zodios-client.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+url="https://api.devcycle.com/swagger.json"
+temp_file=$(mktemp)
+curl -s "$url" -o "$temp_file"
+openapi-zod-client "$temp_file" --with-alias --export-schemas --prettier "prettier.config.js" -o "src/api/zodClient.ts"
+rm "$temp_file"
```

Comment:
maybe the script should be run before we do releases?

Reply:
i'll create a separate ticket for this one too, there are some issues with the openapi-zod-client using outdated types so after you run it you have to manually fix a couple of things :/ so its not ready to be fully automated yet